dev.yml

```diff
@@ -3,17 +3,16 @@
 name: job-iteration
 
 up:
-  - homebrew:
-    - mysql-client:
-        or:        [mysql@5.7]
-  - ruby:
-      version: 2.7.6
-  - isogun
+  - packages:
+      - mysql_client
+  - ruby: 2.7.6
   - bundler
+  - mysql
+  - redis
   - custom:
       name: Create Job Iteration database
-      meet: mysql -uroot -h job-iteration.railgun -e "CREATE DATABASE job_iteration_test"
-      met?: mysql -uroot -h job-iteration.railgun job_iteration_test -e "SELECT 1" &> /dev/null
+      meet: mysql -uroot -h $MYSQL_HOST -P $MYSQL_PORT -e "CREATE DATABASE job_iteration_test"
```

Comment:
I forget, do we actually need to pass these variables, or does `mysql` just read them automatically?

Reply:
We do need to pass them:
```
$ echo $MYSQL_HOST
127.0.0.1
$ echo $MYSQL_PORT
63900
$ mysql -uroot job_iteration_test -e "SELECT 1"
ERROR 2003 (HY000): Can't connect to MySQL server on '127.0.0.1:3306' (61)
$ mysql -uroot -h $MYSQL_HOST -P $MYSQL_PORT job_iteration_test -e "SELECT 1"
+---+
| 1 |
+---+
| 1 |
+---+
```